### PR TITLE
feat: Implemented Incident.io Sink

### PR DESCRIPTION
**Pull request by igorhrcek**
Here is implementation of Incident.io Sink which uses Alert Events V2 to send events to Incidenti.io

### CODE REVIEW

File: src/robusta/core/sinks/incidentio/incidentio_sink.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    TableBlock,
+    KubernetesDiffBlock,
+)
+from robusta.core.sinks.sink_base import SinkBase
+from robusta.core.sinks.sink_config import SinkConfigBase
+from robusta.core.sinks.incidentio.incidentio_sink_params import IncidentioSinkParams
+
+
+class IncidentioSink(SinkBase):
+    def __init__(self, sink_config: SinkConfigBase, registry):
```
Comment (arikalon1) [inline src/robusta/core/sinks/incidentio/incidentio_sink.py:25]: I think the `sink_config` type here should be the concrete type, `IncidentioSinkConfigWrapper`
The base class doesn't have the attribute `incidentio_sink`

File: src/robusta/core/sinks/incidentio/incidentio_sink.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                {
+                    "text": "🔎 Investigate in Robusta",
+                    "href": finding.get_investigate_uri(self.account_id, self.cluster_name),
+                }
+            )
+        else:
+            links.append(
+                {
+                    "text": "🔎 Enable Robusta UI to investigate",
+                    "href": "https://bit.ly/robusta-ui-incidentio",
```
Comment (arikalon1) [inline src/robusta/core/sinks/incidentio/incidentio_sink.py:57]: Do we really want this else clause? 
I checked, and this link `https://bit.ly/robusta-ui-incidentio` doesn't have anything

**Review by arikalon1 — COMMENTED** [review]
thanks for the PR @igorhrcek 

Left 2 minor comments

File: src/robusta/core/sinks/incidentio/incidentio_sink.py (hunk 2 above)
Comment (igorhrcek) [reply src/robusta/core/sinks/incidentio/incidentio_sink.py:57]: Fair point, I removed it.

File: src/robusta/core/sinks/incidentio/incidentio_sink.py (hunk 1 above)
Comment (igorhrcek) [reply src/robusta/core/sinks/incidentio/incidentio_sink.py]: Yeah, you are right, I've updated my implementation.

**Review by arikalon1 — APPROVED** [review]
nice work

Thanks for the contribution @igorhrcek !

**Pull request merged**